From 372d844ba6eaca0b28500fc557857ea18bb25dec Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-11.34-372d844ba6e-20211117T101025Z-avarab@gmail.com>
In-Reply-To: <cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
References: <cover-v3-00.23-00000000000-20211116T114334Z-avarab@gmail.com>
	<cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 25 Mar 2021 11:12:36 +0100
Subject: [PATCH v4 11/34] githooks doc: add a reference to
 uploadpack.packObjectsHook
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Add a reference to the uploadpack.packObjectsHook hook to githooks(5),
for those readers who might be looking for a hook for pack-objects.

I was sure there was such a hook, but finding the docs took me a while
because I started out searching for "pack-objects" in the githooks(5)
documentation. Let's aid future readers with the same confusion.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/githooks.txt | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Documentation/githooks.txt b/Documentation/githooks.txt
index b51959ff941..357103b3c9b 100644
--- a/Documentation/githooks.txt
+++ b/Documentation/githooks.txt
@@ -698,6 +698,13 @@ and "0" meaning they were not.
 Only one parameter should be set to "1" when the hook runs.  The hook
 running passing "1", "1" should not be possible.
 
+SEE ALSO
+--------
+
+The `uploadpack.packObjectsHook` configuration variable can configure
+a hook for linkgit:git-pack-objects[1] to use, but not in a way that
+uses the hook facility discussed above.
+
 GIT
 ---
 Part of the linkgit:git[1] suite
-- 
2.34.0.796.g126c2ced91c

